mantidproject/mantid
# Add help button to plots in workbench.

**Description of work.**
This PR adds a help button to the plots in workbench. It implements this through the static method showPage, in the HelpWindow class, which has now been exposed. Currently the doc pages are blank, but will be filled in before release by Daniel. The help button currently maps the following plots (from FigureType) and help pages:

- 1D Plot  -> plotting/Plot1DHelp.html
- Error bar plot -> plotting/Plot1DHelp.html
- Waterfall -> plotting/PlotWaterfallHelp.html
- Surface -> plotting/Plot3DHelp.html
- Wireframe -> plotting/Plot3DHelphtml
- Colorfil -> plotting/ColorfillPlotHelp.html
- Image -> plotting/ColorfillPlotHelp.html
- Contour -> plotting/ColorfillPlotHelp,html
- Other -> plotting/index.html


**To test:**

1. Build Mantid with qt-help docs. 
2. Load in some data
3. Do all the different types of plots, click help, and check that a relevant doc page is opened.



Fixes #28720. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
